Tools/TestWebKitAPI/Tests/WebKitCocoa/CreateWebArchive.mm

```diff
@@ -468,6 +468,116 @@ function onFramesLoaded() {
     Util::run(&saved);
 }
 
+static const char* htmlDataBytesForBlobURL = R"TESTRESOURCE(
+<body>
+<script>
+function onImageLoad() {
+    window.webkit.messageHandlers.testHandler.postMessage("done");
+}
+async function fetchImage() {
+    response = await fetch("image.png");
+    blob = await response.blob();
+    url = URL.createObjectURL(blob);
+
+    frame = document.createElement("iframe");
+    frame.src = url;
+    document.body.appendChild(frame);
+
+    img = document.createElement('img');
+    img.onload = onImageLoad;
+    img.src = url;
+    document.body.appendChild(img);
+}
+
+function createIframe() {
+    htmlString = "<p>thisisframe</p>";
+    blob = new Blob([htmlString], { type: 'text/html' });
+    url = URL.createObjectURL(blob);
+    frame = document.createElement("iframe");
```

Comment:
Maybe we should test img as well?

Reply:
Already included in test (a few lines above).